src/hooks/use-site-details.tsx

```diff
@@ -267,6 +267,7 @@ export function SiteDetailsProvider( { children }: SiteDetailsProviderProps ) {
 						"Please verify your site's local path directory contains the standard WordPress installation files and try again. If this problem persists, please contact support."
 					),
```

Comment:
What do you think about including `error.message` to the alert?
I feel giving some tips to the user to fix the site will help the overall UX.

Something like:
<img width="1012" alt="Screenshot 2024-10-01 at 13 12 33" src="https://github.com/user-attachments/assets/d3cf3186-5b89-4b27-8cc2-bbb66498360c">

Reply:
Yeah, good point. This could be helpful when the user contacts support and we need to debug a failure.

Reply:
Not sure about removing it as it's part of the error message, it could be helpful when debugging issues in the future. What we could do is adding a title to that part conveying that's the raw error message, so users don't get confused about the content.